velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2943,6 +2943,63 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+// The test for the alias now() is integrated into this test.
+TEST_F(DateTimeFunctionsTest, currentTimestamp) {
+  const auto evaluateCurrentTimestamp =
+      [&](const std::string& functionName,
+          int64_t expectedTimestamp,
+          const std::string& expectedTimeZone) {
+        auto emptyRowVector = makeRowVector(ROW({}), 1);
+
+        auto callExpr = std::make_shared<const core::CallTypedExpr>(
+            TIMESTAMP_WITH_TIME_ZONE(),
+            std::vector<core::TypedExprPtr>{},
+            functionName);
+
+        auto actual = evaluate(callExpr, emptyRowVector);
+        auto actualRow = actual->wrappedVector()->as<RowVector>();
```

Comment:
Generally wrappedVector isn't used like this

auto resultTstz =
                        actual->as<ConstantVector<ComplexType>>()->valueVector()->as<RowVector>();
                auto resultTs = resultTstz->childAt(0)->asFlatVector<int64_t>()->valueAt(0);
                auto resultTz = resultTstz->childAt(1)->asFlatVector<int16_t>()->valueAt(0);

Then you can compare resultTs with expectedTimestamp and the timezones directly without using any vector comparison methods.